pkg/ingester/ingester.go

```diff
@@ -885,6 +888,17 @@ func (i *Ingester) Query(req *logproto.QueryRequest, queryServer logproto.Querie
 		}
 	}
 
+	defer func() {
+		status := "200"
+		if err != nil {
+			status = "400"
```

Comment:
Yeah, I'm not sure what to use here otherwise. The other logging functions with a status field are called from a HTTP handler. Here that's not the case. Perhaps I should use an error field.

Reply:
Yeah it's an open string so feel free to use something more specific like "successful"/"failed".